dev/devicelab/manifest.yaml

```diff
@@ -530,6 +530,13 @@ tasks:
     stage: devicelab_ios
     required_agent_capabilities: ["mac/ios"]
 
+  simple_animation_perf_ios:
+    description: >
+      Measure CPU/GPU usage percentages of a simple animation.
+    stage: devicelab_ios
+    flaky: true
```

Comment:
Please add a comment why this was marked as flaky (e.g. because its new and unproven).

Reply:
Removed flaky.